pages/profile.tsx

```diff
@@ -60,15 +65,25 @@ export const ProfilePage: NextPage = () => {
                 size="lg"
               >
                 <TabList color="black">
-                  <Tab>My Collections</Tab>
-                  <Tab>Profile</Tab>
+                  <Tab
+                    _selected={{ fontWeight: 700 }}
+                    onClick={() => setTabIndex(0)}
```

Comment:
Looking at the Chakra docs for tabs again, it seems like you're supposed to use the `onChange` prop of `Tabs`, not add an `onClick` to individual tabs.